app/controllers/passengers_controller.rb

```diff
@@ -3,6 +3,15 @@ class PassengersController < ApplicationController
   before_action :access_control, only: %i[destroy]
   def new
     @passenger = Passenger.new
+    @doctors_note = DoctorsNote.new
+  end
+
+  def edit
+    if @passenger.doctors_note.present?
```

Comment:
Could also be

```ruby
@doctors_note = @passenger.doctors_note || DoctorsNote.new
```

Is it an issue that `DoctorsNote.new` is not associated with the passenger, whereas `@passenger.doctors_note` would be?